wasm/src/object.c

```diff
@@ -71,6 +74,251 @@ static void __copy_object_elem(opa_object_t *result, opa_value *a, opa_value *b)
     }
 }
 
+opa_object_t *__paths_to_object(opa_value *a)
+{
+    opa_object_t *root = opa_cast_object(opa_object());
+    opa_array_t *paths = opa_cast_array(a);
+
+    for (int i = 0; i < paths->len; i++)
+    {
+
+        opa_object_t *node = root;
+        opa_value *done = opa_boolean(false);
```

Comment:
We don't need an `opa_boolean` here (which is allocated on the heap). An integer would be fine.

Reply:
Done